test/integration-tests/src/test/resources/phase1-testgrid.yaml

```diff
@@ -0,0 +1,48 @@
+# TestGrid Test Configuration Parameters
+# TODO use dash as the separator instead of camelCase.
+version: '0.9'
+emailToList: "wso2testgridphase1@gmail.com,yasassri@wso2.com"
+infrastructureConfig:
+  iacProvider: CLOUDFORMATION
+  infrastructureProvider: AWS
+  containerOrchestrationEngine: None
+  parameters:
+    - JDK : ORACLE_JDK8
+  includes:
+     - CentOS
+     - mysql
+     - ORACLE_JDK8
+  provisioners:
+    - name: prod-wso2-ei-scenario-deployment
+      remoteRepository: "git@github.com:wso2/testgrid.git"
+      remoteBranch: "test"
+      description: Provision Infra for running intg tests
+      scripts:
+        - name: 'Test-Phase-1-Deployment'
+          description: ''
+          type: SHELL
+          phase: CREATE
+          file: test/integration-tests/src/test/resources/test-phase1/infra.sh
+        - name: destroy
+          file: test/integration-tests/src/test/resources/test-phase1/destroy.sh
```

Comment:
We can test whether both these scripts were called correctly, and received the infrastructureConfig, script, and job parameters correctly.

Reply:
Destroy script should be called at the very end. If not, that's an issue. Another thing we can assert.